openmetadata-service/src/test/java/org/openmetadata/service/resources/EntityResourceTest.java

```diff
@@ -1855,11 +1857,10 @@ private void validateChangeEvents(
       }
 
       if (changeEvents == null || changeEvents.getData().size() == 0) {
-        try {
-          Thread.sleep(iteration * 10L); // Sleep with backoff
-        } catch (InterruptedException e) {
-          e.printStackTrace();
-        }
+        ResultList<ChangeEvent> finalChangeEvents = changeEvents;
+        Awaitility.await()
+            .atMost(iteration * 10L, TimeUnit.MILLISECONDS)
```

Comment:
@ssandy83 looks like 10 milliseconds is too short a polling time. We can make this 100 msec and change the total iterations to 10 instead of 25?

Right now we are waiting for 3.25 sec. This will make the loop wait for 5.5 sec (which should be okay)

Reply:
Thanks @sureshms 
This is done